sklearn/ensemble/voting_classifier.py

```diff
@@ -19,7 +19,18 @@
 from ..base import clone
 from ..preprocessing import LabelEncoder
 from ..externals import six
-from ..utils.validation import check_is_fitted
+from ..externals.joblib import Parallel, delayed
+from ..utils.validation import has_fit_parameter, check_is_fitted
+
+
+def _parallel_fit_estimator(estimator, X, y, sample_weight):
+    """Private function used to fit an estimator within a job."""
+    if (sample_weight is not None and
+            has_fit_parameter(estimator, "sample_weight")):
```

Comment:
Really really sorry that I did not catch this. :-(

Yes, we should raise an error at the beginning of fit if `sample_weight` is provided and not all estimators support `sample_weight` as done here (https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/multioutput.py#L81)